client/src/app/employee/employee-detail/employee-detail.component.html

```diff
@@ -0,0 +1,62 @@
+<ng-container *ngIf="employee | async; let employee" class="border-info">
```

Comment:
The unwrapping (`| async`) and displaying of this data _could_ be combined in the same component because the behavior is pretty simple here, however this is usually the boundary that you want to use to separate off your presentational components, which are responsible for laying out state in markup, but only get that state via `@Input()`, not services or `Observable<>`

Reply:
So in this example the `EmployeeComponent` would subscribe the the `Observable<Employee>` and push that Employee object into the `EmployeeDetailComponent`?